app/models/time_record.rb

```diff
@@ -13,4 +14,12 @@ def stop
       spent_time: self.spent_time + time_passed
     )
   end
+
+  private
+
+  def active_task_launched_today
+    if self.assigned_date != Date.today
```

Comment:
As I understand assigned_date is sent be frontend. Date.today is generated by backend. 
So if timezone of frontend is "easter" than server's, no records can be created in this gap. 

For example. I'm in UTC timezone and it's 1.10 23:01. And server is on UTC+1 timezone - here 2.10 00:01
I'm creating record with assigned date 1.10 because it's my current date. Than I am trying to start it but it's 2.10 on server and I will get an error.

It's real?